jib-core/src/main/java/com/google/cloud/tools/jib/cache/Caches.java

```diff
@@ -48,11 +81,17 @@ public Initializer setApplicationCacheDirectory(Path applicationCacheDirectory)
       return this;
     }
 
-    public Caches init() throws CacheMetadataCorruptedException, NotDirectoryException {
-      if (baseCacheDirectory == null || applicationCacheDirectory == null) {
+    public Caches init()
+        throws CacheMetadataCorruptedException, IOException, CacheDirectoryNotOwnedException {
+      if (applicationCacheDirectory == null) {
         throw new IllegalStateException(
-            "Must initialize cache with both base image layer cache directory and application image layer cache directory");
+            "Must initialize cache with an application image layer cache directory");
```

Comment:
I realize I don't know what this error means. It's a error on our side, right?

Reply:
Yeah, this ensures that we must have called `setApplicationCacheDirectory` before calling `init`.

Reply:
I think we can ensure applicationCacheDirectory is set by forcing it as a parameter down on line 99?